src/main/java/at/ac/fhcampuswien/snake/ingameobjects/Food.java

```diff
@@ -0,0 +1,132 @@
+package at.ac.fhcampuswien.snake.ingameobjects;
+
+import at.ac.fhcampuswien.snake.util.Constants;
+import at.ac.fhcampuswien.snake.util.StateManager;
+
+import java.util.Objects;
+
+public class Food {
+
+    private final Position position;
+
+    private String foodType;
+
+    private boolean isSpecialFood;
+
+    public int getScoreValue() {
+        return scoreValue;
+    }
+
+    private int scoreValue;
+
+    private final static int REGULAR_SCORE_VALUE = 1;
+    private final static int SPECIAL_SCORE_VALUE = 3;
+
+    private int specialFoodTimeToLive;
+
+    private final static String[] REGULAR_FOOD_TYPES = new String[]{"1.png", "2.png", "3.png",
```

Comment:
Reorder the modifiers to comply with the Java Language Specification.
```suggestion
    private static final static String[] REGULAR_FOOD_TYPES = new String[]{"1.png", "2.png", "3.png",
```